pop-os/kernelstub
# Kernelstub 3.1.0

This PR merges in Kernlstub 3.1.0 from upstream. A full description of changes can be found in isantop/kernelstub#21

Testing: 
1. Ensure that the `python3-systemd` package is not installed. Removing it will remove the `system76-driver`
2. Run `journalctl -fb` in a terminal to watch the log update live
3. Run `tail -f /var/log/kernlstub.log` in a separate terminal to watch the log update live
4. Run `sudo kernelstub` in a separate terminal and watch for kernelstub log messages in the two logging destinations.
  * There should be none in `journalctl`, and there should be some in `/var/log/kernelstub`
5. Reinstall the `system76-driver` package, and ensure that `python3-systemd` is installed as a dependency. Install it if it isn't.
6. Run `sudo kernelstub` again and check for log messages. 
  * There should be identical log messages in the two outputs.
